schema.py

```diff
@@ -66,8 +66,9 @@ class And(object):
     """
     def __init__(self, *args, **kw):
         self._args = args
-        assert list(kw) in (['error'], [])
+        assert sorted(list(kw)) in (['error', 'schema'], ['error'], [])
         self._error = kw.get('error')
+        self._schema = kw.get('schema', Schema)
```

Comment:
Thanks for your feedback! I think there is a bit problem if you use Schema directly because it's public class but it can not be inherited.

About this code `self.__class__`, I think it's good enough to use it in only `Schema`. I mean about other code, it's good to pass inherited Schema class instead